pkgs/development/haskell-modules/non-hackage-packages.nix

```diff
@@ -23,4 +23,9 @@ self: super: {
   # both are auto-generated by pkgs/development/tools/haskell/haskell-language-server/update.sh
   haskell-language-server = self.callPackage ../tools/haskell/haskell-language-server { };
   hls-ghcide = self.callPackage ../tools/haskell/haskell-language-server/hls-ghcide.nix { };
+
+  ci-info = self.callPackage ../misc/haskell/hasura/ci-info {};
```

Comment:
Maybe give comments how these files were generated, so it's easier to update next time?

Reply:
Will leave your original comments, good idea - thanks!